pkg/k8s/constants.go

```diff
@@ -137,6 +140,11 @@ var (
 		StateMergeIntoSpec,
 		StateAbsentInSpec,
 	}
+	DefaultStateIntoSpecIndicatorAnnotation = FormatAnnotation("default-state-into-spec")
+	DefaultStateIntoSpecIndicatorValues     = []string{
```

Comment:
I noticed the design doc is already updated, are you going to update this PR and simplify the implementation? (removing `default-state-into-spec` annotation).

Reply:
Hi @diviner524 sorry for the confusion. The updated change is in #1149 . Will abandon this PR.